test/amoc_api_status_handler_SUITE.erl

```diff
@@ -43,15 +31,6 @@ returns_up_when_amoc_up_online(_Config) ->
     ?assertEqual(200, CodeHttp),
     ?assertMatch({[{<<"node_status">>, <<"up">>}]}, Body).
 
-
-returns_down_when_api_up_and_amoc_down_offline(_Config) ->
```

Comment:
Why this test case was removed?

Reply:
it doesn't introduce any value. it tests absolutely the same thing as`returns_down_when_api_up_and_amoc_down_online`, but restricts internal implementation.